spec/controllers/places_controller_spec.rb

```diff
@@ -24,118 +26,121 @@
-  }
+  let(:valid_attributes) do
+    skip('Add a hash of attributes valid for your model')
+  end
 
-  let(:invalid_attributes) {
-    skip("Add a hash of attributes invalid for your model")
-  }
+  let(:invalid_attributes) do
+    skip('Add a hash of attributes invalid for your model')
+  end
 
   # This should return the minimal set of values that should be in the session
   # in order to pass any filters (e.g. authentication) defined in
   # PlacesController. Be sure to keep this updated too.
   let(:valid_session) { {} }
 
-  describe "GET #index" do
-    it "returns a success response" do
+  describe 'GET #index' do
+    it 'returns a success response' do
       Place.create! valid_attributes
       get :index, params: {}, session: valid_session
       expect(response).to be_successful
     end
   end
 
-  describe "GET #show" do
-    it "returns a success response" do
+  describe 'GET #show' do
+    it 'returns a success response' do
       place = Place.create! valid_attributes
-      get :show, params: {id: place.to_param}, session: valid_session
+      get :show, params: { id: place.to_param }, session: valid_session
       expect(response).to be_successful
     end
   end
 
-  describe "GET #new" do
-    it "returns a success response" do
+  describe 'GET #new' do
+    it 'returns a success response' do
       get :new, params: {}, session: valid_session
       expect(response).to be_successful
     end
   end
 
-  describe "GET #edit" do
-    it "returns a success response" do
+  describe 'GET #edit' do
+    it 'returns a success response' do
       place = Place.create! valid_attributes
-      get :edit, params: {id: place.to_param}, session: valid_session
+      get :edit, params: { id: place.to_param }, session: valid_session
       expect(response).to be_successful
     end
   end
 
-  describe "POST #create" do
-    context "with valid params" do
-      it "creates a new Place" do
-        expect {
-          post :create, params: {place: valid_attributes}, session: valid_session
-        }.to change(Place, :count).by(1)
+  describe 'POST #create' do
+    context 'with valid params' do
+      it 'creates a new Place' do
+        expect do
+          post :create, params: { place: valid_attributes }, session: valid_session
+        end.to change(Place, :count).by(1)
       end
 
-      it "redirects to the created place" do
-        post :create, params: {place: valid_attributes}, session: valid_session
+      it 'redirects to the created place' do
+        post :create, params: { place: valid_attributes }, session: valid_session
         expect(response).to redirect_to(Place.last)
       end
     end
 
-    context "with invalid params" do
+    context 'with invalid params' do
       it "returns a success response (i.e. to display the 'new' template)" do
-        post :create, params: {place: invalid_attributes}, session: valid_session
+        post :create, params: { place: invalid_attributes }, session: valid_session
```

Comment:
You can try this solution for too long line:
```ruby
post :create, params: { 
  place: invalid_attributes 
}, session: valid_session
```

Reply:
Or if you feel that it will be more readable:
```ruby
params = { place: invalid_attributes }
post :create, params: params, session: valid_session
```